src/index.rs

```diff
@@ -1135,6 +1137,51 @@ impl Index {
         self.enum_qualified_variables.get(&qualified_name.to_lowercase())
     }
 
+    /// Tries to return an enum variant defined within a POU
+    pub fn find_enum_variant_in_pou(&self, pou: &str, variant: &str) -> Option<&VariableIndexEntry> {
+        self.get_enum_variants_in_pou(pou).into_iter().find(|it| it.name == variant)
+    }
+
+    /// Returns all enum variants defined within a POU
+    pub fn get_enum_variants_in_pou(&self, pou: &str) -> Vec<&VariableIndexEntry> {
+        self.enum_qualified_variables
+            .keys()
+            .filter(|key| key.starts_with(&internal_type_name(pou, "").to_lowercase()))
+            .flat_map(|key| self.find_fully_qualified_variable(key))
+            .collect()
```

Comment:
Not a big fan of how filtering works on raw strings here if I'm being honest but I can't think of another way without refactoring how enums are stored in the index. Maybe someone has a better idea that I couldn't think of?